config/settings/development.yml

```
@@ -0,0 +1,2 @@
+min_length_password: 8
+max_length_password: 20
```

```suggestion
length_8: 8
length_20: 20
```
nên đặt tên biến ri nhé
để sau còn tái sử dụng và trực quan nè

---

Dạ chị <3